### ISSUE TITLE
image wont show

### ISSUE THREAD

**Issue by SargTeX** [Krs-Scripts/krs_spawnselector#1]
Image of map wont show

**Comment by Krs-Scripts** [comment]
Send me a photograph, However it seems strange to me, I'm using it and it works

**Comment by RoyaleWind** [comment]
Yeah, I think @Krs-Scripts is using a Discord attachment. @SargTeX, you can open the file in the JS build and replace the image. I also think it's a good idea for you to provide the source so people can help and modify it.

**Comment by Krs-Scripts** [comment]
Tomorrow I'll release that too so you can edit it

**Comment by itsjaCoob** [comment]
that would be nice 💘
